app/src/main/res/drawable/background_fully_transparent.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this file still needed?

Reply:
yes, see https://github.com/streetcomplete/StreetComplete/pull/4642#discussion_r1104892790